### PR TITLE
Add release bot to gauss

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/2856

### CODE REVIEW

File: events/events.go [hunk 1]
```diff
@@ -182,6 +182,7 @@ var environmentProjects = map[Environment][]string{
 		"aws-operator",
 		"node-operator",
 		"passage",
+    "release-bot",
```
Comment (JosephSalisbury) [inline events/events.go]: please fix formatting

Comment (pipo02mix) [reply events/events.go]: This is the result of trying to resolve in github editor (not anymore)

File: events/events.go [hunk 2]
```diff
@@ -182,6 +182,7 @@ var environmentProjects = map[Environment][]string{
 		"aws-operator",
 		"node-operator",
 		"passage",
+                "release-bot",
```
Comment (JosephSalisbury) [inline events/events.go]: There's still a space wrong here

Comment (pipo02mix) [reply events/events.go]: I think this is the +

**Comment by pipo02mix** [comment]
Well my editor should be parse tabs to space. But let me recheck

**Comment by pipo02mix** [comment]
Fixed!

**Pull request merged**